pages/Advanced Queries.md

```diff
@@ -138,6 +138,33 @@ description:: Advanced queries are written with [Datalog](https://en.wikipedia.o
 			- `:Xd-after-ms` - Timestamp of X days after today. **New syntax** `:+Xd-ms`
 			- `:start-of-today-ms` - Timestamp for beginning of today. **New syntax** `:today-start`
 			- `:end-of-today-ms` - Timestamp for end of today. **New syntax** `:today-end`
+	- ### Query parameters and functions
+	  created-at:: 1697057234
+   	  updated-at:: 1697057234
+		- You'll see example queries below that compose concepts expressed in this section.
+		  View [comments in the source](https://github.com/logseq/logseq/blob/master/src/main/frontend/db/query_dsl.cljs/#L22-L48)
+		- #### Page data
```

Comment:
I could see this being useful. Mentioning the 'Show page data' and 'Show block data' commands under `Query Tips` number 5 would be great.